### PR TITLE
suport table

### PR DESCRIPTION
**Pull request by snadn**

### CODE REVIEW

**Comment by stoe** [comment]
@snadn, thanks for the contribution, this is ✨ 

Can you add a test using the `--debug` flag, please, so we can check that the HTML table has the right format/syntax.

Thanks.

**Comment by snadn** [comment]
sorry, i can't understand 'add a test using the --debug flag'.

is these? 
image

but,i think this pr can not merge now, because it is depend on some change in `to-markdown`.

the change i do is 
image

and this pr https://github.com/domchristie/to-markdown/pull/140 do the same. but 'to-markdown' seem is dead

File: .editorconfig [hunk 1]
```diff
@@ -7,3 +7,6 @@ end_of_line = lf
 charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
+
+[*.md]
+insert_final_newline = false
```
Comment (stoe) [inline .editorconfig]: Markdown should always have a final newline.

File: test/expected/test.md [hunk 2]
```diff
@@ -24,4 +24,9 @@ Sometimes it's useful to have different levels of headings to structure your doc
...
 ### This is a third-tier heading
 
-You can use one `#` all the way up to `######` six for different heading sizes.
\ No newline at end of file
+You can use one `#` all the way up to `######` six for different heading sizes.
+
+|Th1|Th2|Th3|
+| --- | --- | --- |
+|Tr1td1|Tr1td2|Tr1td3|
+|Tr2td1|Tr2td2|Tr2td3|
```
Comment (stoe) [inline test/expected/test.md]: Actual output is

```md
### This is a third-tier heading

You can use one `#` all the way up to `######` six for different heading sizes.

| 

Th1

 | 

Th2

 | 

Th3

 |
| 

Tr1td1

 | 

Tr1td2

 | 

Tr1td3

 |
| 

Tr2td1

 | 

Tr2td2

 | 

Tr2td3

 |
```

Could you look into this @snadn, please.

**Comment by stoe** [comment]
@snadn, I was thinking about a test case that checks if the table is in the HTML with the `--debug` flag  and is formatted correctly.

For example:

```js
test.serial('source + table', async t => {
  t.plan(2);

  await execa('./index.js', [t.context.testDocx, '--debug']);

  const html = fs.readFileSync(t.context.testHtml, 'utf8');
  const markdown = fs.readFileSync(t.context.testMd, 'utf8');
  const expected = fs.readFileSync(path.resolve('./test/expected/test.md'), 'utf8');

  t.true(html.indexOf('</td></tr></table>') > 0, 'table missing');
  t.is(markdown, expected, 'table not formatted correctly');
});
```

File: .editorconfig (hunk 1 above)
Comment (snadn) [inline .editorconfig]: I change this because './test/expected/test.md' can't have a final newline

File: test/expected/test.md (hunk 2 above)
Comment (snadn) [inline test/expected/test.md]: this is depend on some change which not merge in `to-markdown` now.

the code in comments https://github.com/stoelzle/mdify-node/pull/6/files#diff-ecf418d6061a5b9417b76093bf2a8085R23 also have effect，but I think do this in `to-markdown` is better.

**Comment by stoe** [comment]
@snadn are you still working on this PR?
Do you need any help?

**Comment by snadn** [comment]
I am so sorry I have already no focus on this PR

**Pull request closed without merging**